File: haskell/fizzbuzz.hs
fizzbuzz :: Integer -> String
fizzbuzz n = case(n, rem n 3, rem n 5) of
	(n, 0, 0) -> "fizzbuzz"
	(n, 0, _) -> "fizz"
	(n, _, 0) -> "buzz"
	(n, _, _) -> show n

main = do 
	putStr $ show $ map fizzbuzz [1..100]

File: javascript/fizzbuzz.js
for (var i=1; i<=100; i++) {
  var fizz = i % 3 === 0 ? 'fizz' : '';
  var buzz = i % 5 === 0 ? 'buzz' : '';

  var output = fizz || buzz ? fizz + buzz : i;

  console.log(output);
}
